src/test/java/com/epam/tc/hw3/util/Property.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.tc.hw3.util;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class Property {
+
+    public String readProperty(String key) {
```

Comment:
A good candidate to become the static method 
And class looks like utility-class, so would be better to make it `final` and revoke the possibility to instantiate it

Reply:
done this remark